src/Refitter/Settings.cs

```diff
@@ -176,4 +176,12 @@ public sealed class Settings : CommandSettings
     [CommandOption("--use-apizr")]
     [DefaultValue(false)]
     public bool UseApizr { get; set; }
+
+    [Description("""
+                 Set the threshold from which to generate a single complex query parameter merging all others. Default is 0 (no merging).
+                 See https://github.com/reactiveui/refit?tab=readme-ov-file#dynamic-querystring-parameters for more information.
+                 """)]
+    [CommandOption("--dynamic-querystring-parameters-threshold")]
+    [DefaultValue(0)]
+    public int DynamicQuerystringParametersThreshold { get; set; }
```

Comment:
I have some mixed feelings about this option @JeremyBP. 

It seems unnecessary to be able to customize the threshold. It would be simpler to have a boolean flag which when set merges query string parameters if there are more than 1.

What do you think?

Reply:
Yep, I totaly agree, mixed feeling too. A feeling about confusing concept of threshold here, that doen't bring value that much. I will refactor that property as a boolean today as suggested. As a boolean, I would name the property like `UseDynamicQuerystringParameters` and the command like `--use-dynamic-querystring-parameters`. The "use" let the user know that it's a boolean obviously, and the "querystring" seems to be a single word everywhere in the Refit doc (but I'm french so... ^^). I have to update the doc with that change too.

Finaly, I'll update the ProjectFileContents class to keep a single .NET 8 target with all in the box, no problem.

Juts tell me if it sounds good to you and expect an updated PR later today.